roles/sft-server/templates/nginx.service.override.j2

```diff
@@ -1,4 +1,19 @@
 [Service]
 LimitNOFILE={{ nginx_rlimit_nofile }}
+
 ExecStartPre=/usr/sbin/nginx -t
+
+# NOTE: redefine reload behaviour to cause configuration management
+#       fail early if Nginx configuration files are flawed
+ExecReload=
 ExecReload=/usr/sbin/nginx -t
+ExecReload=/usr/sbin/nginx -s reload
+
+# NOTE: this is a workaround that - as a side effect - my cause Nginx
+#       end up in an undesired state for some cases
+# ISSUE: https://bugs.launchpad.net/ubuntu/+source/nginx/+bug/1581864
+ExecStartPost=/bin/sleep 0.1
+
+# NOTE: defaults to 'inherit' which leads to error messages not
+#       appearing if nginx.conf is set to 'error_log: stderr'
+StandardError=journal
```

Comment:
Why are we sleeping?   Is it to work around the bug that is linked? (I don't see why) or is it _causing_ the bug that is linked?

Reply:
I'm surprised `inherit` doesn't work    `StandardOutput=` defaults to `journal` and `StandardError=inherit` just means "Connect to the same fd as `StandardOutput`"   Honestly sounds like a systemd bug to me. @flokli  thoughts?

Reply:
This wasn't added in this PR, but I fixed it anyways :)

Reply:
@akshaymankar can you elaborate on why you removed this? The issue that you and I linked closes with `Confirmed → Won't Fix`. I experienced the issue just last week, which was when I introduced the suggested workaround.

@arianvp the line you commented on, was introduced by https://github.com/wireapp/ansible-sft/pull/18. Happy to discuss anything related over there.

Reply:
As I understood from the launchpad thread, this is marked `Won't Fix` for the ubuntu package only. It was fixed in the upstream according to [this comment](https://bugs.launchpad.net/ubuntu/+source/nginx/+bug/1581864/comments/31) from over a year ago. 

I could be wrong though. I did run the role and make sure it works. But as it is a race condition problem, I should've probably run this many times, which I didn't, so maybe I could run `systemctl restart nginx` a few times to make sure it works. To be sure, what version of nginx did you face this issue with?

Reply:
`nginx/1.18.0`. [The comment](https://bugs.launchpad.net/ubuntu/+source/nginx/+bug/1581864/comments/31) also seems to refer to the Ubuntu package, no? The role installs from [Nginx directly](https://github.com/wireapp/ansible-sft/blob/develop/roles/sft-server/tasks/install.yml#L59). Did you spot a link to a commit/change in the thread, so we can see that the actual fix was?